package.json

```diff
@@ -152,11 +152,10 @@
     "sass": "^1.53.0",
     "sass-loader": "10.2.1",
     "storybook": "^7.1.0-alpha.1",
-    "stylelint": "15.10.1",
-    "stylelint-config-recommended": "^7.0.0",
-    "stylelint-formatter-pretty": "^3.1.0",
-    "stylelint-order": "^5.0.0",
-    "stylelint-scss": "^4.2.0",
+    "stylelint": "16.3.1",
+    "stylelint-config-standard-scss": "^13.0.0",
```

Comment:
Using [this package](https://github.com/stylelint-scss/stylelint-config-recommended-scss/blob/master/index.js) instead of the other recommended config helped remove other packages.

This SCSS config included `postcss-scss`, `stylelint-config-recommended`, and `postcssScss` (instead of `postcss-scss`) and I could remove all three of those package in favor of adding this one.